friendshipper/src-tauri/tauri.conf.json

```diff
@@ -91,7 +95,16 @@
 			}
 		},
 		"security": {
-			"csp": null
+			"csp": {
```

Comment:
Could you describe what this is doing?

Reply:
Actually this might not be necessary anymore. Let me do some testing. Originally I was popping the OIDC login into the friendshipper window directly, which requires some CSP modification. Lemme get back to you.

Reply:
confirmed unnecessary